package/MDAnalysis/coordinates/XDR.py

```diff
@@ -148,6 +149,10 @@ def __init__(
             itself is that of the sub system.
         refresh_offsets : bool (optional)
             force refresh of offsets
+        dt : float (optional)
+            timestep in MDAnalysis units to load trajectory with
+            If dt is None, the time is taken from the xdr file
+            Else, the time is set to dt * frame
```

Comment:
Please check that this renders correctly in the docs. Please also use markup as follows for arguments and for Python names:
```reST
If `dt` is ``None`` ... else the time is set to `dt` * frame
```